lib/screens/dji_shop/screens/shopping_cart_screen/widgets/quantity.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_animation/infrastructure/theme/theme_extension.dart';
+
+class Quantity extends StatelessWidget {
```

Comment:
try to think how to develop single reusable widget for vertical and horizontal Quantity